lighthouse-core/scripts/cleanup-LHR-for-diff.js

```diff
@@ -23,9 +23,21 @@ writeFileSync(filename, cleanAndFormatLHR(data), 'utf8');
  * @return {string}
  */
 function cleanAndFormatLHR(lhrString) {
+  /** @type {LH.Result} */
   const lhr = JSON.parse(lhrString);
-  delete lhr.configSettings.auditMode;
-  delete lhr.timing;
+
+  // Set to predictable and typical value
+  lhr.configSettings.auditMode = false;
+
+  // Set timing values, which change from run-to-run, to predicable values
+  lhr.timing.total = 12345.6789;
+  lhr.timing.entries.forEach((entry, i) => {
+    // @ts-ignore
+    entry.duration = 100;
+    // @ts-ignore
+    entry.startTime = 100 * i + i; // 1ms gap between them
```

Comment:
if we add new entries this is going to cause some noise. What about based on length of the name or some kind of hash of the name or something?

Reply:
k. behold our new predictable timing trace! 

![image](https://user-images.githubusercontent.com/39191/47586464-28644280-d915-11e8-9424-7d963b0a550a.png)